zhaquirks/__init__.py

```diff
@@ -35,9 +36,17 @@ class LocalDataCluster(CustomCluster):
 
     async def read_attributes_raw(self, attributes, manufacturer=None):
         """Prevent remote reads."""
-        attributes = [types.uint16_t(a) for a in attributes]
-        values = [self._attr_cache.get(attr) for attr in attributes]
-        return values
+        records = [
+            foundation.ReadAttributeRecord(
+                attr, foundation.Status.UNSUPPORTED_ATTRIBUTE, foundation.TypeValue()
```

Comment:
Why are we doing this?

Reply:
Oh, are you establishing this as the default in case there is nothing in the cache?

Reply:
Yes, exactly.